README.md

```diff
@@ -49,7 +49,7 @@ A note about the gamer method. If you already know the gamers xuid you can use t
 | Method | Value | Optional | Short Description |
 |---       |---    |---              |---       |
 | `get(method=<string>)`        | string  | `term=<string>` | API calls. |
-| `send_message(message=<string>, xuids=[])` | string, list | n/a | Send a message to the following xuids |
+| `send_message(message=<string>, gamers=[])` | string, list | n/a | Send a message to the following xuids |
```

Comment:
The documentation is actually correct now that I look at it, this is also what XboxAPI expects for the messages endpoint. The code should be updated to use `xuids` instead of `gamers`:

```python
  def send_message(self, message=None, xuids=None):
    ''' Send a message given a list of xuids '''
    payload = {}
    if message is None:
      raise ValueError('A message is required!')
    if xuids is None or not hasattr(xuids, 'append'):
      raise TypeError('List was not given!')
    payload['to'] = xuids
    payload['message'] = message
    return self.client.api_post('messages', payload)
```